File: zuzubibi/old/1월/1월 26일/과일 장수.py
def solution(k, m, score):
    answer = 0
    score.sort(reverse=True)
    for i in range(0, len(score), m):
        tmp = score[i:i+m]
        
        if len(tmp) == m:
            answer += min(tmp) * m
    return answer

# 8
print(solution(3,4, [1, 2, 3, 1, 2, 3, 1]))

# 33
print(solution(4,3,	[4, 1, 2, 2, 4, 4, 4, 4, 1, 2, 4, 2]))

File: yht0827/2월/2월 17일/과일장수.java
import java.util.*;

class Solution {
	public int solution(int k, int m, int[] score) {
		int answer = 0;
		Arrays.sort(score);

		for (int i = score.length; i >= m; i -= m) {
			answer += m * score[i - m];
		}
		return answer;
	}
}